src/server/utils/prepare.js

```diff
@@ -10,21 +10,43 @@ var exec = require('child_process').exec,
  * @return {Promise}
  */
 function execCordovaPrepare(projectRoot, platform) {
-    var deferred = Q.defer();
+    return retryAsync(getExecCordovaPrepareImpl(projectRoot, platform), 2);
+}
+
+function getExecCordovaPrepareImpl(projectRoot, platform) {
+    return function () {
+        var deferred = Q.defer();
 
-    log.log('Preparing platform \'' + platform + '\'.');
+        log.log('Preparing platform \'' + platform + '\'.');
 
-    exec('cordova prepare ' + platform, {
-        cwd: projectRoot
-    }, function (err, stdout, stderr) {
-        if (err) {
-            deferred.reject(err || stderr);
+        exec('cordova prepare ' + platform, {
+            cwd: projectRoot
+        }, function (err, stdout, stderr) {
+            if (err || stderr) {
```

Comment:
For what it's worth, the time that I saw this happen to me the error message was actually in stdout, as well as returning a nonzero error code and thus populating `err`